### DISCUSSION TITLE
HELP: Hero covers nav dropdown

### DISCUSSION THREAD

**Discussion by Radicitus** [saadeghi/daisyui#2096] [Q&A] [answered]
Hello!

I have an issue where the nav dropdown is covered by the hero component, and I have had similar issues with a lot of DaisyUI components so I'm not sure what I'm doing wrong.

![Screenshot 2023-07-05 at 16 34 50](https://github.com/saadeghi/daisyui/assets/45127818/8a75674f-c3e2-4d84-b196-1885ce1dc34a)

Here is the code:
// Layout.tsx
![Screenshot 2023-07-05 at 16 43 55](https://github.com/saadeghi/daisyui/assets/45127818/cc493e52-f281-4d02-abc8-e1a98914c879)

// Nav.tsx
![Screenshot 2023-07-05 at 16 44 53](https://github.com/saadeghi/daisyui/assets/45127818/6033767b-b6d9-4447-84a3-eb9616efa44f)

// Hero.tsx
![Screenshot 2023-07-05 at 16 45 08](https://github.com/saadeghi/daisyui/assets/45127818/d4fe1ef8-878e-487a-ac47-be6c985c81e5)

**Comment by saadeghi** [comment, marked as answer]
For dropdown, use a z-index Tailwind CSS class like `z-[1]` or `z-[10]` or anything. I shouldn't and I can't decide for the menu z-index because whether it should be on top of something or under something, depends on the content (for example someone may want it to go on top of a navbar or someone may want it to go under a sticky navbar.) So it's not a good decision for daisyUI to set a z-index value by default.

> I have had similar issues with a lot of DaisyUI components

What are the other issues?

**Reply by Radicitus** [reply]
Hey man!

Thank you so much for this, I had no idea about z-indexes as I'm a new developer. This makes so much more sense now, I really appreciate your help! It definitely makes sense that I shouldn't let DaisyUI set the index for me.

After discovering this, there's no other issues :)
